packages/playwright-core/src/server/browserContext.ts

```diff
@@ -515,7 +515,7 @@ export abstract class BrowserContext extends SdkObject {
     };
     const originsToSave = new Set(this._origins);
 
-    const collectScript = `(${storageScript.collect})((${utilityScriptSerializers.source})(), ${this._browser.options.name === 'firefox'})`;
+    const collectScript = `(${storageScript.collect})((${utilityScriptSerializers.source})(), ${this._browser.options.name === 'firefox'}, ${Boolean(process.env.PLAYWRIGHT_SKIP_INDEXEDDB)})`;
```

Comment:
Let's make it a proper API option.

Reply:
done in https://github.com/microsoft/playwright/pull/34650/commits/2116f5f290560f20677f896657e218d85f76e33f